molgenis-security-core/src/main/java/org/molgenis/security/core/SecureIdGenerator.java

```diff
@@ -0,0 +1,19 @@
+package org.molgenis.security.core;
+
+public interface SecureIdGenerator
+{
+	/**
+	 * Generates a secure identifier.
```

Comment:
Can you document what makes an identifier secure? Should a secure identifier be unique?